### DISCUSSION TITLE
[Feature] Admin-Defined Mobile Defaults for Users

### DISCUSSION THREAD

**Discussion by coltenkrauter** [immich-app/immich#12327] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### Feature Request Summary
As an admin of an Immich instance, I want the ability to define default settings for mobile clients connected to my server. This would allow me to optimize the experience for users like family members, who might not be aware of the ideal configurations suited to my specific server setup and internet bandwidth. For instance, I would like to enable the default option for `using the embedded preview during thumbnail generation` to prevent issues like dark images from [RAW photos](https://github.com/immich-app/immich/discussions/9547#discussioncomment-10548329).

### Use Case
Many users, especially non-technical ones, may not understand how settings like thumbnail handling, RAW image previews, or bandwidth usage can affect their experience when backing up and viewing photos on a mobile device. As the administrator, I have a much better idea of what the optimal settings are for the given hardware and network conditions. By being able to preset defaults for these users:
- I can prevent issues like incorrect image rendering (e.g., dark photos from RAW files).
- I can set sensible defaults that balance server load and user experience, like managing how thumbnails are handled and limiting data usage.

### Proposed Solution:
Immich could provide a section in the `admin/system-settings` page to configure default settings for mobile apps. These defaults would apply to new users but could be overridden by users if they choose.
- **Granular Control** – Admins could apply these defaults globally or to specific user groups like family members.

---

Thanks for the discussion, keep rocking! 🎸

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Closing in favour of the more general #13651.
